techniques/applications/aptPackageManagerSettings/4.0/aptPackageManagerSettings.st

```diff
@@ -122,7 +122,7 @@ bundle agent check_apt_settings
     &APT_URL:{apt_url |"apt_url[&i&]" string => "&apt_url&";
 }&
 
-    &APT_COMPONENT_MAIN, APT_COMPONENT_CONTRIB, APT_COMPONENT_NON_FREE, APT_COMPONENT_RESTRICTED, APT_COMPONENT_UNIVERSE, APT_COMPONENT_MULTIVERSE:{main, contrib, nonfree, restricted, universe, multiverse |"apt_components[&i&]" string => "&main& &contrib& &nonfree& &restricted& &universe& &multiverse&";
+    &APT_COMPONENT_MAIN, APT_COMPONENT_CONTRIB, APT_COMPONENT_NON_FREE, APT_COMPONENT_RESTRICTED, APT_COMPONENT_UNIVERSE, APT_COMPONENT_MULTIVERSE:{main, contrib, nonfree, restricted, universe, multiverse |"apt_components[&i&]" string => "&main& &contrib& &nonfree& &restricted& &universe& &multiverse& &APT_COMPONENT_OTHER&";
```

Comment:
i'm pretty sure this syntax is invalid
it should be

&APT_COMPONENT_MAIN, APT_COMPONENT_CONTRIB, APT_COMPONENT_NON_FREE, APT_COMPONENT_RESTRICTED, APT_COMPONENT_UNIVERSE, APT_COMPONENT_MULTIVERSE, APT_COMPONENT_OTHER:{main, contrib, nonfree, restricted, universe, multiverse, other |"apt_components[&i&]" string => "&main& &contrib& &nonfree& &restricted& &universe& &multiverse& &other&";